src/test/java/core/basesyntax/MyTest.java

```diff
@@ -0,0 +1,88 @@
+package core.basesyntax;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class MyTest {
+    private User user;
+    private String login;
+    private int loginLength;
+    private String password;
+    private int age;
+    private int passwordLength;
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        login = user.getLogin();
+        password = user.getPassword();
+        age = user.getAge();
+        loginLength = user.getLogin().length();
+        passwordLength = user.getPassword().length();
+    }
+
+    @Test
+    void checkNotNullUser() {
+        if (user == null) {
+            throw new MyUncheckedException("User is null");
+        }
+    }
```

Comment:
The method `checkNotNullUser` should use `assertNotNull(user)` instead of manually throwing `MyUncheckedException`. This aligns with the checklist's recommendation to use assertions for test validations.